xbmc/pvr/windows/GUIWindowPVRChannels.cpp

```diff
@@ -124,9 +124,11 @@ bool CGUIWindowPVRChannels::Update(const std::string &strDirectory, bool updateF
   bool bReturn = CGUIWindowPVRBase::Update(strDirectory);
 
   /* empty list for hidden channels */
-  if (m_vecItems->Size() == 0 && m_bShowHiddenChannels)
```

Comment:
I think this whole method is not needed anymore (@xhaggi?) since it should not be possible to set `m_bShowHiddenChannels` to true unless there actually are some hidden channels to display.